code-of-conduct.md

```diff
@@ -82,20 +82,20 @@ A direct message ("DM") is a private message to one or more other members. Sendi
 
 ## Not For Profit (Mostly)
 
-This community is hyper-allergic to unwanted commercial behavior. This is not a place for prominent commercial activities such as recruiting, lead generation, marketing, and other solicitation, except in channels dedicated to that purpose. This includes starting market research conversations outside of #market-research. If you join this community to take value rather than contribute, the community will quickly notice and react.
+Some parts of this community are averse to unwanted commercial behavior. This is not the best place for prominent commercial outreach activities such as recruiting, lead generation, marketing, and other solicitation, except in channels dedicated to that purpose. This includes performing unambiguously commercial market research conversations for paid products outside of #market-research. If it's unclear: ask yourself whether the conversation will at least as valuable to other members of the community than it is for you. If you join this community primarily to take value rather than make balanced contributions, the community will quickly notice and react.
```

Comment:
The other day we were talking about the "crazy" autoresponse, which is currently

> Using the words crazy, insane, or bonkers is considered by some to be insensitive to sufferers of mental illness...

and I suggested removing "considered by some", because it was important that the bot responses represent the position of _RLS_, not of "some people". 

I think we have the same problem here. The code of conduct can't reflect the position of "some people", it is the rules everyone agrees to and so it needs to be clear and direct.

The community, as a whole, _is_ hyper-allergic to commercial behavior, and it's good to warn people up front that the reaction to a misplaced post is going to be strong.

All of the edits here make expected conduct less clear to readers who don't know what's expected already, which is the opposite of what a _code of conduct_ should do.

Reply:
> This includes performing unambiguously commercial market research conversations for paid products outside of #market-research.

I think that "for paid products" opens a loophole for free products.

An alternate wording to consider:

This includes performing market research for paid or commercial products outside of #market-research.

Reply:
> hyper-allergic

@wlonkly Part of my reasoning for changing this is, exactly like "crazy"; as-is doesn't seem to be very sensitive to people who are hyper-allergic to real-life things i.e. if they consume them and do not have medical attention: they will die.

> The code of conduct can't reflect the position of "some people", it is the rules everyone agrees to and so it needs to be clear and direct.

I agree. Based on the conversations we've had, though, I think the existing rules as stated are _not_ agreed to by everyone.

> All of the edits here make expected conduct less clear to readers who don't know what's expected already, which is the opposite of what a code of conduct should do.

Is there no single word change you agree with here? For example, in the case of someone who started a thread, after being nudged to post more on their blog, whether "what might you find interesting on my blog?" in a public channel would be OK in this case rather than "market research".



> I think that "for paid products" opens a loophole for free products.

@michaelmj This was not my intention, I was more thinking about e.g. open source products, personal blogs, etc.

Reply:
Ah, I see. I don't have a strong opinion on avoiding "hyper-allergic" as a metaphor, but I think what you proposed significantly weakens it. "The community has strong opinions about..." or something?

>  I think the existing rules as stated are not agreed to by everyone.

Sorry, I mean that we tell new members that they must _agree_ to the code of conduct in order to participate. Because of that they need to be unambiguous. (I don't mean internal dis-_agree_-ments about what the CoC _should_ say.)

Hrm, maybe it might help to summarize the "Based on some discussion from #rls-culture" in the PR here, so that we can separate "what are the goals of the change" from "does the language support the goals"?

Reply:
> I don't have a strong opinion on avoiding "hyper-allergic" as a metaphor, but I think what you proposed significantly weakens it. "The community has strong opinions about..." or something?

I like this 👍🏻 

> Sorry, I mean that we tell new members that they must agree to the code of conduct in order to participate. 

Ah, gotcha, sorry!

> Hrm, maybe it might help to summarize the "Based on some discussion from #rls-culture" in the PR here, so that we can separate "what are the goals of the change" from "does the language support the goals"?

Yes, good idea, will do this 👍🏻